### DISCUSSION TITLE
Welcome to tm4e Discussions!

### DISCUSSION THREAD

**Discussion by mickaelistria** [eclipse/tm4e#441] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by angelozerr** [comment]
TM4e fixes several problem with on enter rules and I think this feature is very helpful for XML/HTML and other languages used in WWD.

It should be nice to have a release of TM4e.

Do you think it is possible to create this release?

**Reply by mickaelistria** [reply]
+1 from me for a release ASAP. @sebthom @akurtakov What do you think?

**Reply by akurtakov** [reply]
Fine with me.

**Reply by mickaelistria** [reply]
TM4E 0.6.1 is now released.

**Reply by angelozerr** [reply]
Thanks!
